pkg/epforwarder/epforwarder.go

```diff
@@ -50,7 +51,8 @@ var passthroughPipelineDescs = []passthroughPipelineDesc{
 		intakeTrackType:        "dbmmetrics",
 		// raise the default batch_max_concurrent_send from 0 to 10 to ensure this pipeline is able to handle 4k events/s
 		defaultBatchMaxConcurrentSend: 10,
-		defaultBatchMaxContentSize:    20e6,
+		defaultBatchMaxContentSize:    pkgconfig.DefaultBatchMaxContentSize,
```

Comment:
In the most extreme scenarios, our individual `dbm-metrics` events can approach the max payload size, so with this change we may actually start dropping events that were previously making it through. 

In DBM we don't need a separate single message size limit. We care only about the max payload size. 

So I think this change could be simplified by by sticking to only the one `batch_max_payload_size` setting? So we'd effectively rename `batch_max_content_size` to `batch_max_payload_size` and keep a config alias for `batch_max_content_size` -> `batch_max_payload_size` in case some agents have overridden the defaults?

Reply:
> I agree that's a better name

Actually, I now remember why I chose "content size" as the name for that setting instead of "payload size". It's because the payloads are compressed before sending. So if we rename the setting to `batch_max_payload_size` then that also isn't quite true if "payload" refers to the thing actually being sent over the wire.

Reply:
Thanks for the feedback. Ive made this change much simpler by just setting the default max content size to 5MB. Knowing that the intake will truncate (but not drop) messages larger than 1MB, we don't need to enforce this on the agent.